src/routes/accommodation.route.js

```diff
@@ -239,4 +239,131 @@ router.get('/:subjectID/comments', verifyToken.headerToken, commentMiddleware.va
+ *      summary: user should be to rate an accommodation
+ *      tags: [Accommodations]
+ *      parameters:
+ *       - name: accomodationId
+ *         in: path
+ *         description: rate that specific accomodation
+ *         required: true
+ *         type: string
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *      requestBody:
+ *        required: true
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/schemas/rate'
+ *      responses:
+ *        "200":
+ *           description: rate create successfully
+ *
+ * components:
+ *    schemas:
+ *      rate:
+ *        type: object
+ *        required:
+ *          - rate
+ *        properties:
+ *          rate:
+ *            type: integer
+ */
+
+router.post('/:accomodationId/rate', verifyToken.headerToken, verifyUser, AccomodationMiddleware.checkValidAccomodationRates, AccomodationMiddleware.checkIfUserBookedThatAccomodation, Accommodation.rateAccomodation);
+/**
+ * @swagger
+ *
+ * /accommodations/{accomodationId}/rate:
+ *    patch:
+ *      summary: user should be to update accommodation rate
+ *      tags: [Accommodations]
+ *      parameters:
+ *       - name: accomodationId
+ *         in: path
+ *         description: rate that specific accomodation
+ *         required: true
+ *         type: string
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *      requestBody:
+ *        required: true
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/schemas/rate'
+ *      responses:
+ *        "200":
+ *           description: rate create successfully
+ *
+ * components:
+ *    schemas:
+ *      rate:
+ *        type: object
+ *        required:
+ *          - rate
+ *        properties:
+ *          rate:
+ *            type: integer
+ */
+
+router.patch('/:accomodationId/update-rate', verifyToken.headerToken, verifyUser, AccomodationMiddleware.checkValidAccomodationRates, AccomodationMiddleware.checkIfUserBookedThatAccomodation, Accommodation.updateAccomodationRate);
+/**
+ * @swagger
+ *
+ * /accommodations/{accomodationId}/rate:
+ *    get:
+ *      summary: user should get rate of an accommodation
+ *      tags: [Accommodations]
+ *      parameters:
+ *       - name: accomodationId
+ *         in: path
+ *         description: id of the accommodation
+ *         required: true
+ *         type: string
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *      responses:
+ *        "200":
+ *           description: rate create successfully
+ */
+router.get('/:accomodationId/rate', verifyToken.headerToken, verifyUser, Accommodation.getAccommodationRate);
```

Comment:
/ratings, these are individual ratings

Reply:
I changed it in every route now it's /ratings.